cmake/Modules/LocalSvt.cmake

```diff
@@ -1,4 +1,4 @@
-set(AVIF_SVT_GIT_TAG "v2.3.0")
+set(AVIF_SVT_GIT_TAG "38bc2f92f0360f8280858f40f3229f796c70b9c5")
```

Comment:
Did you forget to undo the changes to this file?

Reply:
That is just to make sure the CI passes with 3.0.0.
If we merge the .c file, it will work with 2.3.0 but not with HEAD (which still has not bumped its version), that is why I suggest to wait for upstream to to bump their version, even for an RC